aptos-move/move-examples/post_mint_reveal_nft/sources/minting.move

```diff
@@ -110,7 +122,13 @@ module post_mint_reveal_nft::minting {
 
     /// SourceToken stores the token_data_id of the source certificate token.
     struct SourceToken has key {
-        token_data_id: TokenDataId,
+        source_collection_name: String,
+        // this is the base name for the source token. when minting a source token, the name will be
+        // {source_token_name} {counter}
```

Comment:
Can you mention this at the top of the file as well as part of "Here's what things look like if you use this contract" (along with other important characteristics)?